src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -50,25 +49,31 @@ internal sealed class EventPipeController
         private const string ConfigKey_MultiFileSec = "MultiFileSec";
 
         // The default set of providers/keywords/levels.  Used if an alternative configuration is not specified.
-        private static readonly EventPipeProviderConfiguration[] DefaultProviderConfiguration = new EventPipeProviderConfiguration[]
+        private static EventPipeProviderConfiguration[] DefaultProviderConfiguration => Defaults.ProviderConfiguration;
+
+        // Inner static type to hold static initalization to avoid creating a static .cctor that's activated in the startup path
+        private static class Defaults
         {
-            new EventPipeProviderConfiguration("Microsoft-Windows-DotNETRuntime", 0x4c14fccbd, 5, null),
-            new EventPipeProviderConfiguration("Microsoft-Windows-DotNETRuntimePrivate", 0x4002000b, 5, null),
-            new EventPipeProviderConfiguration("Microsoft-DotNETCore-SampleProfiler", 0x0, 5, null),
-        };
+            internal static readonly EventPipeProviderConfiguration[] ProviderConfiguration = new EventPipeProviderConfiguration[]
```

Comment:
Caching does not seems to be worth it either. It is only used when we have parsed the configuration file (a lot of allocations) and the file has no explicitly specified configuration (rare).

I would change this to a method that creates a fresh array every time.

Reply:
Done